### DISCUSSION TITLE
Paypal Fees too high for donation

### DISCUSSION THREAD

**Discussion by ghost** [AnySoftKeyboard/AnySoftKeyboard#3249] [General]
Hi, I would like to make a donation to ASK but the paypal fees are too high! I don't want to support greedy financial capitalists.
I would much prefer to donate to a bank account for ASK
I would suggest opening an account with WISE which lets you recieve transfers with bank details for different areas (you can have EU account details, US account details, Australian etc etc).

**Comment by menny** [comment]
yeah, I know... I understand. There is also an option to go through GitHub sponsors (which does not have fees). Since AnySoftKeyboard is not a financial organization we can not open a bank account under that name. And I can not open a foreign bank account as I'm a US person. I'm really sorry about that.

**Comment by mistanoize** [comment]
Hope it's ok to ask here...

Is there somewhere to make a one-off donation other than PayPal (I too exclude PayPal)?

**Reply by menny** [reply]
I think it is possible to do that using GitHub sponsor, but not sure.

Thank you for your support

**Comment by RusneroV** [comment]
What about creating a Monero and Bitcoin wallet for donations? I always send a few bucks to developers that accept Monero for donations, it makes it easy for me. You should try it out, if you don't like it in the end, just remove it.
